src/main/java/com/synopsys/integration/alert/channel/slack/descriptor/SlackDescriptor.java

```diff
@@ -36,12 +36,13 @@
 
     public static final String SLACK_LABEL = "Slack";
     public static final String SLACK_URL = "slack";
+    public static final String SLACK_DESCRIPTION = "This page allows you to configure the Slack for Alert.";
```

Comment:
There's nothing to configure on the Global Configuration UI for Slack.  The descriptions for the channels probably need to needs to be different going forward.  Maybe:
"This channel is responsible for distributing messages to Slack."  
For example email might be:
"This channel is responsible for distributing messages to an Email SMTP server."